daemon/container.go

```diff
@@ -612,9 +616,22 @@ func (container *Container) RestoreNetwork() error {
 	return nil
 }
 
+func (container *Container) cleanupiptables() {
+	var (
+		container_ip     = container.NetworkSettings.IPAddress
+		resolv_conf_path = container.ResolvConfPath
+	)
+	for _, net := range container.hostConfig.Netout {
+		bridge.SetupNetoutIptables("-D", net, container_ip, resolv_conf_path)
```

Comment:
why not use a cmd type(enum) such as "AddRule" or "DeleteRule" ?
using "-D" and "-I"  as parameters are misunderstanding. we need to read the whole function to know the meaning of it.